### DISCUSSION TITLE
An internal error occurred while this artifact was being saved.

### DISCUSSION THREAD

**Discussion by michaelgrigoryanwork** [fastlane/fastlane#21114] [Q&A]
Hello everyone, yesterday Xcode Cloud was working well with Fastlane, but today builds are failed with message
**An internal error occurred while this artifact was being saved.**
Does anyone know is it about Fastlane or Xcode Cloud is not working normally?

Thanks in advance.
<img width="469" alt="Screen Shot 2023-03-07 at 19 20 12" src="https://user-images.githubusercontent.com/56847423/223475318-4d564636-6307-4832-b0cf-54d80c685c7b.png">

**Comment by sof9816** [comment]
Same happening to me, first it was complaining on the ci_script, I fixed it by updating gem, and now it is not uploading  the artifacts, I read some threads on apple site, and most of em are saying it is problem caused by apple it self, let's see if someone could confirm this or have a solution.
